packages/react/src/components/Icon/FontIcon.tsx

```diff
@@ -40,29 +42,42 @@ export const getIconContent = memoizeFunction(
 export const FontIcon: React.FunctionComponent<IFontIconProps> = props => {
   const { iconName, className, style = {} } = props;
   const iconContent = getIconContent(iconName) || {};
-  const { iconClassName, children, fontFamily } = iconContent;
+  const { iconClassName, children, fontFamily, mergeImageProps } = iconContent;
 
   const nativeProps = getNativeProps<React.HTMLAttributes<HTMLElement>>(props, htmlElementProperties);
   const containerProps =
     props['aria-label'] || props['aria-labelledby'] || props.title
       ? {
-          role: 'img',
+          role: mergeImageProps ? undefined : 'img',
         }
       : {
           'aria-hidden': true,
         };
 
+  let finalChildren = children;
+
+  if (mergeImageProps) {
+    if (typeof children === 'object' && typeof children.props === 'object') {
```

Comment:
Could you maybe do this only if `props.title` is defined? Seems like it could be expensive.